src/components/videoCard/VideoCard.jsx

```diff
@@ -33,17 +45,17 @@ const VideoCard = ({ video }) => {
           />
           <div className="video-runtime text-sm text-white">{runTime}</div>
         </div>
-        <div className="videoText-container ">
-          <div className="text-sm video-alt flex-row">
-            <span>
-              <AiOutlineFire color="#9c9c9c" />
-              {views}
-            </span>
-            <span className="video-uplodaded">{dateUploaded}</span>
-          </div>
-          <p className="video-title fw-500 text-base">{textOverflow(title)}</p>
-          <p className="text-sm  video-alt"> {creator}</p>
+      </Link>
+      <div className="videoText-container ">
+        <div className="text-sm video-alt flex-row">
+          <span>
+            <AiOutlineFire color="#9c9c9c" />
```

Comment:
why the use of color and not class?

Reply:
i have used default color prop in the icon provider so giving it a class here will not work.